pandas/core/indexes/datetimes.py

```diff
@@ -1937,8 +1937,10 @@ def tz_convert(self, tz):
                      mapping={True: 'infer', False: 'raise'})
     def tz_localize(self, tz, ambiguous='raise', errors='raise'):
         """
-        Localize tz-naive DatetimeIndex to given time zone (using
-        pytz/dateutil), or remove timezone from tz-aware DatetimeIndex
+        Convert aware to naive.
```

Comment:
Localize a tz-naive DatetimeIndex to tz-aware DatetimeIndex

Reply:
@jreback 
can I add this line in extended summary :
tz_localize() takes a naive datetime object and interprets it as if it is in that timezone. It does not move the time to another timezone. tz_localize function helps to switch b/w time zone aware and time zone unaware objects.

Reply:
interprets -> sets it in that timezone with the current timestamps.

> tz_localize function helps to switch b/w time zone aware and time zone unaware objects.

not sure this last line is necessary

Reply:
@jreback If it is give more clearity then It is okey other wise I can remove ??????